### PR TITLE
add ngo model #issue3

### LINKED ISSUE #3: Create ngo model
## Tasks:

- Create `Ngo` model

### PR DESCRIPTION
**Pull request by BerkAkipek**
Fixes #3

### CODE REVIEW

File: src/models/ngo.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  },
+  provider: {
+    type: String,
+  },
+  providerId: {
+    type: String,
+  },
+});
+
+module.exports = mongoose.model('ngo', ngoSchema);
```
Comment (muslimomar) [inline src/models/ngo.js:49]: let's name the `ngo` model as `Ngo`.

File: src/models/ngo.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+
+const ngoSchema = new mongoose.Schema({
+  ngoName: {
```
Comment (muslimomar) [inline src/models/ngo.js:4]: can you rename it as `name` only.

File: src/models/ngo.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    type: String,
+    required: true,
+  },
+  webSite: {
```
Comment (muslimomar) [inline src/models/ngo.js:8]: Let's make it `website`

File: src/models/ngo.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  },
+  publishedEvents: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Event',
+    },
+  ],
+  password: {
+    type: String,
+    required: true,
```
Comment (muslimomar) [inline src/models/ngo.js:28]: `password` shouldn't be required, because we might use social auth like google auth

**Review by muslimomar — CHANGES_REQUESTED** [review]
Great work

**Comment by BerkAkipek** [comment]
Hello Muslim,
I edit model in from your comments.

**Pull request merged**